### PR TITLE
Translate doc snippets md files from en to de

### LINKED ISSUE #3774: [FEATURE REQUEST] Translate to snippet docs to German
### Description of feature request, idea

Trying to make PR review snippets more inclusive by having multiple languages

### Add (multiple) screenshot links (if applicable)

_No response_

### Add any other context about the feature request here

_No response_

### PR DESCRIPTION
**Pull request by YuriDevAT**
I could not find an issue to be closed by this. I am sorry if I have overseen it.

closes #3774 

#### Description of the task to be completed?
I translated md files from English to German

#### Any background context you want to provide?
For now, I translated it kind of formally (like, "you" in Sir, and not "you" like a "friend". The English version does not seem that formal. I can change this any time to casual as well.

#### Questions
Should I do the same in Japanese? Of course, I would have my translation reviewed by my Japanese Tandem Partner 😄

### CODE REVIEW

File: docs/snippets/de/github-action-failing.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## GitHub Actions sind fehlgeschlagen
+
+### TLDR;
+
+Vielen Dank für den PR. Eine GitHub Action ist fehlgeschlagen. Bitte prüfen Sie
+den Output der Action, um mehr über den Fehler zu erfahren.
+
+Falls Sie Fragen haben, zögern Sie nicht, diese im EddieHub Discord Channel unter
+zu stellen http://discord.eddiehub.org
```
Comment (schmelto) [inline docs/snippets/de/github-action-failing.md:9]:
```suggestion
Falls du Fragen haben solltest, kannst du diese gerne im [EddieHub Discord Channel](http://discord.eddiehub.org) stellen.
```

File: docs/snippets/de/github-action-failing.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## GitHub Actions sind fehlgeschlagen
+
+### TLDR;
+
+Vielen Dank für den PR. Eine GitHub Action ist fehlgeschlagen. Bitte prüfen Sie
+den Output der Action, um mehr über den Fehler zu erfahren.
```
Comment (schmelto) [inline docs/snippets/de/github-action-failing.md:6]:
```suggestion
Vielen Dank für den PR. Eine GitHub Action ist fehlgeschlagen. Bitte prüfe
den Output, um mehr über den Fehler zu erfahren.
```

**Comment by YuriDevAT** [comment]
Thank you so much for your suggestions, Tom. And sorry for the inconvenience. As suggested in my PR I would have changed the formal to casual terms if needed 😅

**Comment by eddiejaoude** [comment]
Thank you both so much 🔥 

Oops I didn't create an issue 🤦‍♂️

**Review by eddiejaoude — APPROVED** [review]
Thank you so much 👍

**Pull request merged**